overlord/devicestate/devicestate_test.go

```diff
@@ -766,7 +766,7 @@ version: gadget
 func (s *deviceMgrSuite) TestDoRequestSerialErrorsOnNoHost(c *C) {
 	privKey, _ := assertstest.GenerateKey(testKeyLength)
 
-	nowhere := "http://nowhere.invalid"
+	nowhere := "http://so-very-invälid.com"
```

Comment:
I really don't like using a .com here,  afaict (testing on artful)

nowhere.nowhere.test  

should also work